code/app/src/main/java/com/example/goldencarrot/views/CancelledUsersAdapter.java

```diff
@@ -6,14 +6,12 @@
 import android.widget.TextView;
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
-import com.example.goldencarrot.R;
 
+import com.example.goldencarrot.R;
 import com.example.goldencarrot.data.model.user.User;
-
```

Comment:
Why do we need a cancelledUserArrayAdapter?

shouldn't this be a UserArrayAdapter that we can reuse for Accepted waiting list, rejected waiting list etc..?

Reply:
FYI I made the cancelled UserAdapter before the User array adapter was created so thats why it is there. I did tell the group I already made an array adapter but I guess someone made a new one. I can just get rid of it and update OrganizerCancelled view

Reply:
please rename this to UserArrayAdapter 

it is better to generalize so that we can reuse the code for other lists cancelled, accepted, etc..

Hooriya and James are also working on something similar, please make sure you are not writing the same code twice with them